templates/install_pipenv.jj2

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = 'https://pypi.python.org/simple'
+verify_ssl = true
+name = 'pypi'
+
+[requires]
+python_version= '3.6'
+
+[packages]
+{% for dependency in dependencies: %}
+    {% if '=' in dependency %}
+        {{dependency}}
+    {% else %}
+        {{dependency}} = "*"
+    {% endif %}
+{% endfor %}
+
```

Comment:
this trailing blank line needs to be removed.

Reply:
Done